tests/spread/integration/openjdk-8-jre-headless/task.yaml

```diff
@@ -0,0 +1,14 @@
+summary: Integration tests for openjdk-8-jre-headless
+
+environment:
+  SLICE/core: "core"
+
+execute: |
+  # Test different slice installations
+  echo "SLICE=${SLICE}"
+  rootfs="$(install-slices busybox_bins openjdk-8-jre-headless_${SLICE})"
+  (cd ${rootfs}/bin && for applet in $(${rootfs}/bin/busybox --list); do \
+      ln -s busybox $applet ; \
+    done)
+  cp smoke.sh "${rootfs}/"
+  chroot ${rootfs} busybox sh /smoke.sh
```

Comment:
This feels like a lot of unnecessary setup...
 Why not

```
cd ${choot}
mkdir -p proc/self
for java in `find usr/lib/jvm -name java`; do
    ln -sf /${java} proc/self/exe
    chroot . ${java} -version
done
```

Reply:
We need to set up applets because chroot contains only busybox executable.

Reply:
yes but why do you need busybox at all? shouldn't my snippet above work?

Reply:
We need to install coreutils and bash for it.